ideascube/mixins.py

```diff
@@ -11,6 +11,36 @@
 from ideascube.search.models import Search
 
 
+class OrderableViewMixin:
+
+    ORDERS = []
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        if self.ORDERS:
+            context['orders'] = self.ORDERS
```

Comment:
Did you know? http://reinout.vanrees.org/weblog/2014/05/19/context.html

That is, you could just remove this line, and in the template you could use `{{ view.ORDERS }}` directly.